tests/sentry/integrations/slack/test_unfurl.py

```diff
@@ -188,14 +191,15 @@ def test_unfurl_discover_previous_period(self, mock_generate_chart):
     @patch("sentry.integrations.slack.unfurl.discover.generate_chart", return_value="chart-url")
     def test_unfurl_discover_multi_y_axis(self, mock_generate_chart):
         min_ago = iso_format(before_now(minutes=1))
+        unique_proj = self.create_project()
         self.store_event(
-            data={"fingerprint": ["group2"], "timestamp": min_ago}, project_id=self.project.id
+            data={"fingerprint": ["group2"], "timestamp": min_ago}, project_id=unique_proj.id
         )
         self.store_event(
-            data={"fingerprint": ["group2"], "timestamp": min_ago}, project_id=self.project.id
+            data={"fingerprint": ["group2"], "timestamp": min_ago}, project_id=unique_proj.id
         )
 
-        url = f"https://sentry.io/organizations/{self.organization.slug}/discover/results/?field=title&field=event.type&field=project&field=user.display&field=timestamp&name=All+Events&project={self.project.id}&query=&sort=-timestamp&statsPeriod=24h&yAxis=count_unique%28user%29&yAxis=count%28%29"
```

Comment:
These giant URLs are going to be hard to maintain. Can you rewrite them with `urlencode()`?

Reply:
I don't think we can this case. There are some repeated parameters: `?field=title&field=event.type&field=project&field=user.display&field=timestamp&name=All+Events&`

Reply:
```py
params = urlencode([
    ("field", "event.type"),
    ("field", "project"),
    ("field", "timestamp"),
    ("field", "title"),
    ("field", "user.display"),
    ("name", "All+Events"),
    ("project", self.project.id),
    ("query", ""),
    ("sort", "-timestamp"),
    ("statsPeriod", "24h"),
    ("yAxis", "count_unique(user)"),
    ("yAxis", "count()"),
])
url = f"{BASE_URL}/organizations/{self.organization.slug}/discover/results/?{params}"
```
Its a little painful but it kinda works

Reply:
Oops I see the change is gone, nvm.